File: cubby/FractionFormatter/src/org/acm/mbaechi/fractions/DoubleFractionRounder.java
package org.acm.mbaechi.fractions;

public class DoubleFractionRounder extends AbstractFractionRounder<Double> {

    DoubleFractionRounder(int fraction) {
        super(fraction);
    }

    @Override
    public Double round(Double number) {
        return Math.round(number / divisor) * divisor;
    }
}

File: cubby/csharp/FractionFormatter/FractionFormatter/DoubleFractionRounder.cs
using System;

namespace FractionFormatter
{
	public class DoubleFractionRounder : AbstractFractionRounder<Double> {

	    public DoubleFractionRounder(int fraction) : base(fraction) {
	    }

	    override public Double round(Double number) {
	        return Math.Round(number / divisor) * divisor;
	    }
	}
}


